mne/io/meas_info.py

```diff
@@ -1477,3 +1484,31 @@ def _empty_info(sfreq):
     info._update_redundant()
     info._check_consistency()
     return info
+
+
+def _force_update_info(info_base, info_target):
+    """Update target info objects with values from info base.
+
+    Note that values in info_target will be overwritten by those in info_base.
+    This will overwrite all fields except for: 'chs', 'ch_names', 'nchan'.
+
+    Parameters
+    ----------
+    info_base : mne.Info
+        The Info object you want to use for overwriting values
+        in target Info objects.
+    info_target : mne.Info | list of mne.Info
+        The Info object(s) you wish to overwrite using info_base. These objects
+        will be modified in-place.
+    """
+    exclude_keys = ['chs', 'ch_names', 'nchan']
+    info_target = np.atleast_1d(info_target).ravel()
+    all_infos = np.hstack([info_base, info_target])
+    msg = 'Inputs must be of type Info. Found type {0}'
+    if not all([isinstance(ii, Info) for ii in all_infos]):
+        raise ValueError(msg.format(type(ii)))
```

Comment:
Please follow our usual pattern of string with %s, and don't make it a separate var